modules/persistent-storage-csi-vsphere-increase-max-vols-per-node-procedure.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-increase-max-vols-per-node_{context}"]
+= Increasing the maximum allowable volumes per node for vSphere
+
+.Prerequisites
+* Access to the {product-title} web console.
+
+* Access to the cluster as a user with the cluster-admin role.
+
+.Procedure
+
+To increase the maximum number of volumes per node for vSphere:
```

Comment:
Could this be moved to line 9 and revised as maybe something like:

```suggestion
Use the following procedure to increase the maximum number of volumes per node for vSphere.
```

That way we have an introductory sentence under the heading.